src/extended/PluginHostChildProcess/main.cc

```diff
@@ -100,13 +100,50 @@ class PluginHostApplication : public JUCEApplication {
 
       deviceManager.initialise(256, 256, savedAudioState, true);
 
+      AudioPluginInstance *instance;
       OwnedArray<juce::PluginDescription> foundPlugins;
-      VST3PluginFormat format;
-      format.findAllTypesForFile(foundPlugins, pluginPath);
 
-      description = foundPlugins[0];
-      AudioPluginInstance *instance = format.createInstanceFromDescription(
+      #if (JUCE_PLUGINHOST_VST3 && (JUCE_MAC || JUCE_WINDOWS))

```

Comment:
I had to use these constants to detect if each plugin format class is defined. I don't know if there's a cleaner way to do it.